### PR TITLE
heal: Add --update-parity flag to change the parity of objects

### PR DESCRIPTION
**Pull request by vadmeste**
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
> One object only?

It does recursively apply to all, the PR title is misleading @klauspost

File: cmd/admin-heal.go [hunk 1]
```diff
@@ -73,6 +73,10 @@ var adminHealFlags = []cli.Flag{
 		Name:  "rewrite",
 		Usage: "rewrite objects from older to newer format",
 	},
+	cli.BoolFlag{
+		Name:  "update-parity",
+		Usage: "Update the parity of an object with the new server parity configuration",
```
Comment (klauspost) [inline cmd/admin-heal.go:78]:
```suggestion
		Usage: "Update the parity of objects with the new server parity configuration",
```

**Review by klauspost — CHANGES_REQUESTED** [review]
I guess this should be changed as well, then.

File: cmd/admin-heal.go [hunk 2]
```diff
@@ -73,6 +73,10 @@ var adminHealFlags = []cli.Flag{
 		Name:  "rewrite",
 		Usage: "rewrite objects from older to newer format",
 	},
+	cli.BoolFlag{
+		Name:  "update-parity",
+		Usage: "Update the parity of objects with the new server parity configuration",
```
Comment (harshavardhana) [inline cmd/admin-heal.go:78]:
```suggestion
		Usage: "Update the parity of object(s) with the new server parity configuration",
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by klauspost — APPROVED** [review]

**Pull request closed without merging**